src/java.base/share/native/libjli/java.c

```diff
@@ -957,6 +984,9 @@ AddOption(char *str, void *info)
     /* In OpenJ9 -Xmso is used to set native stack size instead of -Xss. -Xss is used to
      * set Java thread size only, which is handled in the JVM code.
      */
+    if (JLI_StrCCmp(str, "-Xoptionsfile=") == 0) {
```

Comment:
I think the comment above should be expanded to point out that `-Xmso` might be found in an options file.

Reply:
It seems obvious from the code.

Reply:
Updated the other comment to remove the obvious.

Reply:
Maybe it would help if the function names included "Xmso" (e.g. `parseXmoFile()` would be `parseXmsoFile()`).
The part that's less obvious here, is that as we consider the file named in `-Xoptionsfile=`, we're only paying attention to `-Xmso` options we find.

Reply:
Updated.